.github/workflows/checks.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         uses: ./.github/actions/python-environment
```

Comment:
Please add a brief description to https://github.com/exasol/python-toolbox/issues/392 answering:
* what we want to change
* why we want to make this change